lib/scope.js

```diff
@@ -347,11 +347,12 @@ function define(nockDefs) {
       body = undefined
     }
 
-    //  Response is not always JSON as it could be a string or binary data or
-    //  even an array of binary buffers (e.g. when content is encoded)
+    // Response is not always JSON as it could be a string or binary data or
+    // even an array of binary buffers (e.g. when content is encoded).
     let response
     if (!nockDef.response) {
       response = ''
+      // TODO: Rename `responseIsBinary` to `reponseIsUtf8Representable`.
```

Comment:
Should we start saving def with the new name now and keep looking for both until later?

Reply:
Yea, it's a nice idea, though a lot of logic changed here so I'd rather handle it in a new PR.